### DISCUSSION TITLE
Simplistic Old Duprs

### DISCUSSION THREAD

**Discussion by RimblyWasTaken** [Duprs/Feedback#13] [Ideas]
**What is your feature?**
A clear and concise description of what your feature is
Void terrain and not being able to get points for ranks,
**What platform is this for?**
Example: Discord, Minecraft, etc.
Minecraft
**Detailed explaination**
Give a detailed rundown of what this feature should do
I believe both having overworld terrain and being able to earn points for ranks wasnt healthy for the server. By the end of season 4, when the server was still alive in the sense of the word, ranks were more common than ever and it seemed like over half the player base had a rank, and the majority was ultra for that matter. I believe that being able to buy ranks should be the only way to get ranks, to add that speciality to it. If the point system is to be put in place, I believe it should be very, very much increased due to how common they were from season 4 onwards. Also I believe that the world terrain should be completely voided, as it was within season 1, 2 and 3. This added that sence of rarity to some items. For example I remember back in season 2/3, I had a base that needed grass blocks and spending about 20/30 minutes trying to find it. Also i believe it added a lot more creativity to builds as many bases are based off of terrain generation and without that people have to become a lot more creative with builds and some of the bases that came out of void generation completely outshines the terrain bases by far/
**Why should this be added?**
Give a rundown of what the purpose of this feature is, and who it will benefit
This feature within itself could benefit the entire playerbase as it completely changes the way the playerbase would function. It also made finding bases a lot harder for griefers as theres no terrain or chunk changes to base it off of whihc just increases the want to build for some players and also fuels that drive to grief for those greifers as finding it makes for a better reward. On the topic of rewards, this is what I feel about the point/ad system to ranks. It devalues those people who payed for those ranks to just be joined by people who have a chat spamming mod set for 5 minutes sat in a lobby for an hour and a half. I remember back in the old days, ultra and even legend being a rarity to see in the tab menu. I distinctly remember there only being 1 or 2 at most in a lobby jam packed with 30/35 players and it being really cool. I think personally that having void generation allows for more creativity, more reward and generally better gameplay and having the rank ad system devalues those players who bought those ranks before it became a thing.
**Additional content**
Include any supportive text or imagery so that we can better understand your feature [optional]
I'm an old player, I remember the old players, the old server and the old game. Bring it back, Echo and Bassouli

**Comment by Setloth** [comment]
understood, i do agree that the world should be back to the basics per se; a void world. 

and i see the reasoning for ranks wanting to be a more exclusive selection of people, both of these will be the case, thank you 🙏

**Comment by Setloth** [comment]
Side question for you, how should we do ranks for the server?

What I mean is, there are people who owned ranks on duprs previously, but since we no longer obtain that data of those who had those ranks, what should we do?
The unfortunate course of action that seems most probable now is that we just wont be able to transfer ranks, and we will just supply the ranks we sell and that's that.
Understandably frustrating to those who have previously owned ranks, but that seems to be reality.

**Comment by Rimbly** [comment]
As I see it, I think we should go off of discord as many people such as myself already have ranks and old screenshots. For instance I have many screenshots of old duprs, I might be able to recover some old ranks. Maybe opening a new channel in the discord for people to post their screenshots to help recover ranks. Seeing as they cannot be transferred, this would probably be the most honest and the best proof to give people their roles again

**Comment by Setloth** [comment]
alrighty- so my new vision now is now that we can give people who have a rank (any of them) a legacy rank in the new age, to better consolidate, i don't think trusting screenshots is the best idea, but yes we will use discord ranks as a placeholder for those who had ranks before, i think we will also go through the discord, and get those who have ranks currently and give them legacy, so that we know who is actually a legacy player.

do you think that we should name this rank differently from legacy, to make it a new role or combine it into legacy and have legacy be the role that captures those who currently have a rank.

this way the old players with ranks are rewarded with a rank, and it allows us to build around it

**Comment by RimblyWasTaken** [comment]
I'd say we leave legacy as it is. The whole idea around legacy was for those players who payed for a role which was changed for the better and they were rewarded with a closed, unique rank which no one could ever get again. Also remember, Legacy is just the old VIP, which might make people mad as they would lose so many of the things they paid for. I think there should be a new role for those who paid on the old one, but it shouldnt be legacy. Legacy was a rarity, and it had a closed amount of people who had it. Giving people those roles would make it into just another role and would remove what made legacy special. I suggest maybe redesigning this new role with the abilities of Legend? that way tthe new one isnt as low as VIP but its not as high as ULTRA, middle ground as it seems.

**Comment by Setloth** [comment]
why would they lose anything? - I was saying we could combine those currently owning a rank as the legacy rank, which would stay as it was, nothing would lose permissions.

**Comment by RimblyWasTaken** [comment]
Wait now im confused, so the people who already own a rank would have that rank transferred into legacy?

**Comment by Setloth** [comment]
yeah, that's what i was saying.
for this new age of duprs, those who have a rank (any) on the discord would be filed into a legacy rank, which represents that they have played in a previous age of duprs. 
that rank would then be locked to those individuals

**Comment by Rimbly** [comment]
Ahh yes, that does sound good. I'd say we make a new role for them however

**Comment by Setloth** [comment]
what shall it be named

**Comment by Rimbly** [comment]
Maybe Antiques? Ancient?

**Comment by Setloth** [comment]
it shall be decided by the poll

**Comment by Rimbly** [comment]
Indeed it shall

**Comment by Setloth** [comment]
Discussion has been made of this issue, to allow others to contribute their thoughts on aspects of old duprs being used in the new server

**Comment by Rimbly** [comment]
Ahh good, I'd like to hear from the old community and the new community alike. I feel we can come to a consensus sooner that way
